samples: nrf9160: modem_shell: stored commands could be set to be run after a startup (#8077)
nrfconnect/sdk-nrf: samples/nrf9160/modem_shell/README.rst

----

Running stored commands after startup
=====================================

MoSh command: ``startup_cmd``

> You can store up to 3 MoSh commands to be run after a start/bootup.

Reviewer: ```suggestion
You can store up to three MoSh commands to run on start/bootup.
```

> By default, commands are run after the default PDN context is activated,

Reviewer: Is this meant to be on two lines?

Author: I think it is ok now, not meant to be on 2 lines.

> As a default, commands are run after a default PDN context is getting active,

Reviewer 2: ```suggestion
By default, commands are run after default PDN context is getting active,
```

> As a default, commands are run after a default PDN context is getting active,

Reviewer: ```suggestion
By default, commands are run after the default PDN context is activated,
```

> but optionally can be set to be run N seconds after a bootup.

Reviewer: ```suggestion
but can be set to run N seconds after bootup.
```

Examples
--------

> * An example of starting periodic location acquiring after a LTE has been connected with both cellular and gnss including sending location to cloud:

Reviewer 2: ```suggestion
* Starting periodic location acquiring after LTE has been connected with both cellular and GNSS including sending location to cloud:
```

Reviewer: Edied to add a comma. 

Is this specifically to nRF Cloud or any cloud?
